queries/scala/highlights.scm

```diff
@@ -96,6 +93,9 @@
 (generic_function
   function: (identifier) @function.call)
 
+(interpolated_string_expression
+  interpolator: (identifier) @function.call)
```

Comment:
Sting Interpolation is not a function call in other languages

Reply:
Yea, so I went back and forth about this. The main reason I chose this is that under the hood in Scala this is actually represented as a `def`. For example you can see this when hovering on the actual interpolator in Scala:

<img width="352" alt="Screenshot 2023-01-18 at 09 25 49" src="https://user-images.githubusercontent.com/13974112/213120885-0aa8aea6-e87f-440a-afe9-cfe40d8d11b5.png">

It's a macro call, but I thought `function.call` was the most fitting here. Would you suggest something else?

Reply:
a it's the interpolator not the dollar style variable interpolation!